index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>SMART ON FHIR STARTER APP</title>
+   
+   <script src="./lib/fhir-client.js"></script>

```

Comment:
Move the JS script tag's to end of html body. Usually html head should only load css since loading external JS might stall the UI.